<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsWSettingsConnection</class>
 <widget class="QWidget" name="tlinsWSettingsConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>System event monitor (keyboard and joystick)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="sysMonitorAutoConnectDisconnect">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Connect/disconnect automatically</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxAddresses">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="2">
         <widget class="QLineEdit" name="urlDeviceEngineAsynch"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Device engine address</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="urlDeviceEngine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="urlAstroEngine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>System service</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QComboBox" name="deviceEngineAsyncAuthType">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="urlSystemService"/>
        </item>
        <item row="5" column="3">
         <widget class="QComboBox" name="systemServiceAuthType"/>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="astroEngineAsyncAuthType">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="astroEngineAuthType">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Device engine address asynch</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Astro engine address asynch</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QComboBox" name="deviceEngineAuthType">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Astro engine address</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="urlAstroEngineAsynch"/>
        </item>
        <item row="3" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="commonAddress">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="1">
         <widget class="tlinsWDecimalValue" name="astroEngineAsyncPort" native="true"/>
        </item>
        <item row="5" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="tlinsWDecimalValue" name="astroEnginePort" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Astro engine port</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Astro engine asynch port</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Device engine asynch port</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="tlinsWDecimalValue" name="systemServicePort" native="true"/>
        </item>
        <item row="3" column="1">
         <widget class="tlinsWDecimalValue" name="deviceEnginePort" native="true"/>
        </item>
        <item row="4" column="1">
         <widget class="tlinsWDecimalValue" name="deviceEngineAsyncPort" native="true"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>System service port</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Device engine port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Address IP</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="commonAddressIp"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="useCommonAddressIp">
     <property name="text">
      <string>Use the same address</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tlinsWDecimalValue</class>
   <extends>QWidget</extends>
   <header location="global">tlinswdecimalvalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useCommonAddressIp</sender>
   <signal>clicked(bool)</signal>
   <receiver>tlinsWSettingsConnection</receiver>
   <slot>useCommonIp(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>750</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>useCommonIp(bool)</slot>
 </slots>
</ui>
